template/.github/workflows/publish_dev_artifacts.yml.j2

```diff
@@ -45,4 +46,23 @@ jobs:
           -u 'github:${{ secrets.NEXUS_PASSWORD }}'
           --upload-file "./$(find  target/helm/ -name *.tgz)"
           "${{ env.REPO_HELM_DEV_URL }}/"
-
+      - name: Clean old manifests
+        run: mkdir -p deploy/manifests && rm -r $(find deploy/manifests -maxdepth 1 -mindepth 1 -not -name kustomization.yaml)
```

Comment:
This will fail if the folder deploy/manifests exists.

Reply:
I don't see how, `mkdir -p` doesn't fail if the directory already exists. Or did you mean a different part of the command?

Reply:
Ah `rm -r` can file if the find command doesn't return anything! I'll fix that